options.js

```diff
@@ -1,21 +1,45 @@
 let page = document.getElementById('colorSelector');
 
-const googleGreen = "#009688";
-const googleAquaBlue = "#00BBD3";
-
 const input = document.getElementById("colorInput");
-let submitButton = document.getElementById("submitButton");
+getExtensionColor().then(color => setExtensionColor(color));
+getColorMenu().then(colorOptions => buildButtons(colorOptions));
 
-getExtensionColor().then(color => {
-  input.value = color;
-  submitButton.style.backgroundColor = color;
-});
+let addButton = document.getElementById("addButton");
```

Comment:
It's better to put all the buttons declarations on top, before the function calls

Reply:
Ok, changed.